styles.css

```diff
@@ -0,0 +1,153 @@
+* {
+  margin: 0;
+  background-color: #ffff;
+  text-align: center;
+  font-style: normal;
+}
+
+nav {
+  display: block;
+  position: absolute;
+  left: 0;
+  top: 0;
+  width: 100%;
+}
+
+nav a {
+  text-decoration: none;
+  width: 90px;
+  height: 28px;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+  position: absolute;
+  left: 24px;
+  top: 10px;
+  color: #28352f;
+}
+
+nav svg {
+  position: absolute;
+  top: 16px;
+  right: 28px;
+  color: #28352f;
+  cursor: pointer;
+}
+
+.intro {
+  display: flex;
+  position: absolute;
+  top: 112px;
+  left: 24px;
+  flex-direction: column;
+}
+
+.intro h1 {
+  width: 327px;
+  height: 192px;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+}
```

Comment:
[OPTIONAL] I've noticed that you used a fixed width for your whole project. Please take into account that, although this is the width displayed on the design, you shouldn't use a fixed width for your projects (it is just the width of the device used for the screenshot). I will explain: 

The idea of Responsive Web Design is that your webpage should adapt itself to any device/window size that opens it. In other words, it is responsive. This means that using a fixed width for your sections will break this responsiveness, causing a static view to be formed. So, what can you do about it? Simple, use percentages! Instead of `width: 327px`, use `width: 100%`. This will make your project stretches to any window size that you can imagine 😉. If you need a smaller section, use a lower percentage. If you really need to use a fixed width, keep in mind that it may break the responsiveness of your webpage 😄! 

_But, my mobile version looks horrible stretched to a big window 😢._ 

Don't worry about this! There is a thing called [media query](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries) that will allow you to add different styling to different window sizes in the future. Then, you can edit a mobile version and a desktop version for your apps, depending on the window size 😄. But for now, you should focus on the mobile version 😉. 

With a fixed width, even on mobile screens, it doesn't fit correctly. Check your project on an larger mobiles:
![image](https://user-images.githubusercontent.com/42799579/205131090-9016d5c9-55d1-4664-930d-7f9913f58161.png)

If you are interested to learn more about it, check [this link](https://www.w3schools.com/html/html_responsive.asp) and this [youtube video](https://www.youtube.com/watch?v=VQraviuwbzU). 

Overall, you did a pretty good job! The fonts are correct and the design is flawless. Good luck and keep coding 😃!